planning/autoware_objects_of_interest_marker_interface/src/objects_of_interest_marker_interface.cpp

```diff
@@ -82,7 +82,7 @@ ColorRGBA ObjectsOfInterestMarkerInterface::getColor(
       return coloring::getAmber(alpha);
     case ColorName::RED:
       return coloring::getRed(alpha);
-    case ColorName::GRAY:
+    case ColorName::GRAY:  // NOLINT
```

Comment:
Instead of marking it as "NOLINT" you can move this `GRAY` case to the top of the switch chain. And it shouldn't complain.

If you read here: https://clang.llvm.org/extra/clang-tidy/checks/bugprone/branch-clone.html

It explains it in detail.

Specifically this case:

> ```cpp
> switch (ch) {
> case 'a':
> case 'A':
>   return 10;
> case 'b':
> case 'B':
>   return 11;
> default:
>   return 10;
> }
> ```
> Here the check does not warn for the repeated return 10;, which is good if we want to preserve that 'a' is before 'b' and default: is the last branch.

Reply:
Thank you for the advice.
I have confirmed that the error no longer occurs and have fixed it.